Makefile

```diff
@@ -83,7 +83,7 @@ build: git_hooks reformat-diff libp2p_helper
 
 build_archive: git_hooks reformat-diff
 	$(info Starting Build)
-	ulimit -s 65532 && (ulimit -n 10240 || true) && dune build src/app/archive/archive.exe --profile=$(DUNE_PROFILE)
+	ulimit -s 65532 && (ulimit -n 10240 || true) && dune build src/app/archive/archive.exe --profile=testnet_postake_medium_curves
```

Comment:
Can you change the profiles in the CI jobs instead? This will clobber the build artifacts when using this during development.

Reply:
I think the change in 4d26dac should do